ci_support/run_docker_build.sh

```diff
@@ -49,13 +49,22 @@ source run_conda_forge_build_setup
 yum install -y libX11-devel libXt-devel libXext-devel chrpath libXrender-devel gtk2-devel dbus-devel libSM-devel libICE-devel
 
 
-# Install conda-build 2.x.
-conda install --yes --quiet conda-build=2
-conda info
+# Embarking on 3 case(s).
+    set -x
+    export CONDA_PERL=5.20.3.1
```

Comment:
Why not? Perl is set in `build`.

Admittedly I'm only able to see this part of the diff now. So if you are getting multiple Perls that is an issue. ( https://github.com/conda-forge/conda-smithy/issues/415 ) Would appreciate your thoughts on how we solve that.